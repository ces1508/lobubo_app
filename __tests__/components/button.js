import React from 'react'
import Button from '../../src/components/button'

import renderer from 'react-test-renderer'

test('render correctly', () => {
  const tree = renderer.create(<Button text='aceptar' />).toJSON()
  expect(tree).toMatchSnapshot()
})
test('render whit custom width', () => {
  const tree = renderer.create(<Button text='aceptar' width='200' />).toJSON()
  expect(tree).toMatchSnapshot()
})