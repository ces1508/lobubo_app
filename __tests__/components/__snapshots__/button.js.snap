// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly 1`] = `
<View
  accessible={true}
  isTVSelectable={true}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#807bff",
      "borderRadius": 5,
      "flexDirection": "row",
      "height": undefined,
      "justifyContent": "center",
      "opacity": 1,
      "padding": 15,
      "width": undefined,
    }
  }
>
  <Text
    style={
      Array [
        Object {
          "color": "#fff",
        },
        undefined,
      ]
    }
  >
    aceptar
  </Text>
</View>
`;

exports[`render whit custom width 1`] = `
<View
  accessible={true}
  isTVSelectable={true}
  onResponderGrant={[Function]}
  onResponderMove={[Function]}
  onResponderRelease={[Function]}
  onResponderTerminate={[Function]}
  onResponderTerminationRequest={[Function]}
  onStartShouldSetResponder={[Function]}
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#807bff",
      "borderRadius": 5,
      "flexDirection": "row",
      "height": undefined,
      "justifyContent": "center",
      "opacity": 1,
      "padding": 15,
      "width": "200",
    }
  }
>
  <Text
    style={
      Array [
        Object {
          "color": "#fff",
        },
        undefined,
      ]
    }
  >
    aceptar
  </Text>
</View>
`;
